Docs/source/theory/boundary_conditions.rst

```diff
@@ -0,0 +1,303 @@
+.. _theory-bc:
+
+Boundary conditions
```

Comment:
Title case?
```suggestion
Boundary Conditions
```

Reply:
Not sure. Seems like multiple case conventions are used. I personally prefer lower case for all words except the first one, proper names such as "Lorentz", and other such exceptions.

Reply:
I like title case, but I also miss the diverse spelling of words from German :D